xblock/reference/plugins.py

```diff
@@ -171,3 +130,42 @@ def __set__(self, xblock, value):
         We don't support this until we figure out what this means
```

Comment:
@pmitros The comment "until we figure out what this means" could certainly use clarification.  Also `__set__`, while standard Python, is one of its most advanced and obscure features. It seems very reasonable to me to ask for more explanation here.